src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocumentElement.cs

```diff
@@ -361,6 +361,9 @@ static string ToString (WindowRotationAnimation value)
 
 		static string ToString (string value, ICustomAttributeProvider provider, IAssemblyResolver resolver)
 		{
+			//NOTE: this could be an AndroidResource
```

Comment:
This fix doesn't make sense to me; how/why are we hitting this codepath in the first place?

The only place where this `ToString(string, ICustomAttributeProvider, IAssemblyResolver)` overload is called is from when the "type" is `System.Type`:

https://github.com/xamarin/xamarin-android/blob/a3267e34d35a457f11ec1bb4565c350002f8c332/src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocumentElement.cs#L193

`ApplicationAttribute.NetworkSecurityConfig` is *not* of type `Type`; it is of type `string`.  We thus should instead be hitting *this* converter:

https://github.com/xamarin/xamarin-android/blob/a3267e34d35a457f11ec1bb4565c350002f8c332/src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocumentElement.cs#L184

But we aren't hitting `value.ToString()`.  Why aren't we?

We're hitting `ToString (value.ToString (), p, r)` because the `ApplicationAttribute.NetworkSecurityConfig` mapping information *overrides* the default to say "this value is a `System.Type`!":

https://github.com/xamarin/xamarin-android/blob/a3267e34d35a457f11ec1bb4565c350002f8c332/src/Xamarin.Android.Build.Tasks/Mono.Android/ApplicationAttribute.Partial.cs#L168

Looks like this was overlooked in commit 36993b3a1dbd8732fd47eec74992d6aad5b6575f.

I believe that the appropriate fix is to *remove* `ApplicationAttribute.Partial.cs` line 168, so that it is treated as a string.  I see no reason for this property to *ever* be handled as if it were a System.Type.